src/snapred/backend/recipe/algorithm/DiffractionSpectrumWeightCalculator.py

```diff
@@ -28,76 +28,46 @@ def PyInit(self):
             MatrixWorkspaceProperty("WeightWorkspace", "", Direction.Output, PropertyMode.Mandatory),
             doc="The output workspace to be created by the algorithm",
         )
-        self.declareProperty("DetectorPeakIngredients", defaultValue="", direction=Direction.Input)
         self.declareProperty("DetectorPeaks", defaultValue="", direction=Direction.Input)
 
         self.setRethrows(True)
         self.mantidSnapper = MantidSnapper(self, __name__)
 
-    def chopIngredients(self):
-        pass
-
-    def unbagGroceries(self):
-        self.inputWorkspaceName = self.getPropertyValue("InputWorkspace")
-        self.weightWorkspaceName = self.getPropertyValue("WeightWorkspace")
-
-    def validateInputs(self) -> Dict[str, str]:
-        errors = {}
-        waysToGetPeaks = ["DetectorPeaks", "DetectorPeakIngredients"]
-        definedWaysToGetPeaks = [x for x in waysToGetPeaks if not self.getProperty(x).isDefault]
-        if len(definedWaysToGetPeaks) == 0:
-            msg = """
-            Must specify either DetectorPeaks,
-            OR DetectorPeakIngredients to generate the peaks
-            """
-            errors["DetectorPeaks"] = msg
-            errors["DetectorPeakIngredients"] = msg
-        elif len(definedWaysToGetPeaks) == 2:
-            logger.warn(
-                """Both a list of detector peaks and ingredients were given;
-                the list will be used and ingredients ignored"""
-            )
-        return errors
```

Comment:
Note: this validation step is no longer needed, as all peaks must come from the DetectorPeaks property